src/components/FooterWrapper.js

```diff
@@ -43,15 +43,15 @@ const FooterWrapper = ({ children, alwaysShow, fixedHeight }) => {
   })
 
   const popoutItem = ({ link, displayName }) => {
-    return [
+    return h(Fragment, [
```

Comment:
@marctalbott I noticed this console warning that was introduced in https://github.com/DataBiosphere/terra-ui/pull/2965. Sorry that I didn't catch it on the review.

````
react_devtools_backend.js:3973 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `FooterWrapper`. See https://reactjs.org/link/warning-keys for more information.
    at div
    at FooterWrapper (http://localhost:3000/static/js/src_appLoader_js-data_image_svg_xml_3Csvg_xmlns_27http_www_w3_org_2000_svg_27_viewBox_270_0_1-d7b3bc.chunk.js:5496:23)
    at HeroWrapper (http://localhost:3000/static/js/src_appLoader_js-data_image_svg_xml_3Csvg_xmlns_27http_www_w3_org_2000_svg_27_viewBox_270_0_1-d7b3bc.chunk.js:15996:32)
    at SignIn
    at AuthContainer (http://localhost:3000/static/js/src_appLoader_js-data_image_svg_xml_3Csvg_xmlns_27http_www_w3_org_2000_svg_27_viewBox_270_0_1-d7b3bc.chunk.js:759:23)
    at ErrorWrapper (http://localhost:3000/static/js/src_appLoader_js-data_image_svg_xml_3Csvg_xmlns_27http_www_w3_org_2000_svg_27_viewBox_270_0_1-d7b3bc.chunk.js:5046:208)
    at LocationProvider (http://localhost:3000/static/js/src_appLoader_js-data_image_svg_xml_3Csvg_xmlns_27http_www_w3_org_2000_svg_27_viewBox_270_0_1-d7b3bc.chunk.js:40029:24)
    at Main